packages/coze-uniapp/src/types.ts

```
@@ -0,0 +1,51 @@
+  private events: Record<string, Function[]> = {};
+
+  on(eventName: string, callback: Function) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+    this.events[eventName].push(callback);
+    return this;
+  }
+
+  off(eventName?: string, callback?: Function) {
+    if (!eventName) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)